src/bci_build/package/templates/badges.j2

```diff
@@ -1,6 +1,10 @@
 {% if image.eula == 'sle-bci' -%}
 ![Redistributable](https://img.shields.io/badge/Redistributable-Yes-green)
 {%- endif %}
-{% if not image.is_opensuse and image.support_level | string != 'l3' -%}
+{%- if not image.is_opensuse and image.support_level | string != 'l3' -%}
```

Comment:
I know that this is unrelated, but don't we want to highlight l3 support on these images?